.github/workflows/docs.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Grab gtk-rs LOGO
         if: ${{ (github.event_name == 'push' && github.ref == 'refs/heads/master') || github.event_name == 'release' }}
         run: |
-          sudo dnf install wget -y
+          apt-get install wget -y
```

Comment:
No `sudo`? Also: didn't pay attention to that...

Reply:
Somehow the ubuntu image doesn't require one because you're sudoer by default

Reply:
Also `apt` instead of `apt-get` is the modern way, but doesn't make a difference really